frontend/packages/pipelines-plugin/src/utils/__tests__/pipeline-test-data.ts

```diff
@@ -290,3 +290,76 @@ export const mockRunDurationTest = [
   pipelineTestData[PipelineExampleNames.WORKSPACE_PIPELINE].pipelineRuns[DataState.SUCCESS],
   pipelineTestData[PipelineExampleNames.CLUSTER_PIPELINE].pipelineRuns[DataState.SUCCESS],
 ];
+
+export const pipelineWithWorkspace: PipelineKind[] = [
```

Comment:
It will be good if we reuse the existing pipeline test data from `pipelines-plugin/src/test-data/pipeline-data.ts`. This file exclusively contains different pipeline test data including one for `WORKSPACE_PIPELINE`. WDYT?

Reply:
using existing PL